tests/test_box_Box.py

```diff
@@ -439,6 +439,49 @@ def test_plot_2d(self):
         box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
         box.plot()
 
+    def test_compute_distances_2d(self):
+        box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
+        testpoints = np.array([[-2.2, -1.3, 0], [0., 0., 0.], [0., 0., 0.]])
+        testpoints_query = np.array(
+            [[-0.5, -1.3, 0], [0.2, 0., 0.], [0., -0.4, 0.]])
+        distance = box.compute_distances(testpoints, testpoints_query)
+        npt.assert_allclose(distance, [0.3, 0.2, 0.4], rtol=1e-6)
+
+        # 1 dimensional array
+        distance = box.compute_distances(testpoints[0], testpoints_query[0])
+        npt.assert_allclose(distance, 0.3, rtol=1e-6)
+
+        with self.assertRaises(ValueError):
+            box.compute_distances(testpoints[:-1], testpoints_query)
+
+    def test_compute_distances_3d(self):
+        box = freud.box.Box(2, 3, 4, 1, 0, 0)
+        testpoints = np.array([[-2.2, -1.3, 2], [0., 0., 1.], [0., 0., 0.1]])
+        testpoints_query = np.array(
+            [[-0.5, -1.3, 2.], [0.2, 0., 1.], [0., -0.4, 0.1]])
+        distance = box.compute_distances(testpoints, testpoints_query)
+        npt.assert_allclose(distance, [0.3, 0.2, 0.4], rtol=1e-6)
+
+    def test_compute_all_distances_2d(self):
+        box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
+        testpoints = np.array([[0., 0., 0.]])
+        testpoints_query = np.array(
+            [[0.2, 0., 0.], [0., -0.4, 0.], [1., 1., 0.]])
+        distance = box.compute_all_distances(testpoints, testpoints_query)
```

Comment:
Pluralize array name.
```suggestion
        distances = box.compute_all_distances(testpoints, testpoints_query)
```